main.go

```diff
@@ -1,13 +1,24 @@
 package main
 
 import (
+	_ "embed"
 	"fmt"
 	"os"
 
 	"github.com/0xPolygon/polygon-edge/command/util"
+	"github.com/0xPolygon/polygon-edge/params"
 	"github.com/mitchellh/cli"
 )
 
+var (
+	//go:embed LICENSE
+	license string
+)
+
+func init() {
+	params.SetLicense(license)
+}
+
```

Comment:
Do we need this init?
Why can't we set this in the actual `main` method?

Reply:
Why do you want to set in `main` method? This is a initial flow and have `license` on the beginning, then we can call `SetLicense` in `init` function

Reply:
I'm suggesting we move away from these `init` methods

Reply:
Sorry, I don't get it. How do you do instead of this? Putting the call in main function? I don't know what the difference is

Reply:
Explicitly calling, `init` methods just hide away the code being executed

Reply:
Resolving as it was fixed in [ac0a2e6](https://github.com/0xPolygon/polygon-edge/pull/407/commits/ac0a2e637a241829b9bda839bf60c20b1a9b0b1a)